Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -196,7 +196,7 @@ def output_total(formulae)
         unbottled_formulae = 0
 
         formulae.each do |f|
-          next if f.bottle_specification.tag?(@bottle_tag)
+          next if f.bottle_specification.tag?(@bottle_tag, no_older_versions: true)
 
           unbottled_formulae += 1
         end
```

Comment:
```suggestion
        unbottled_formulae = formulae.count do |f|
          f.bottle_specification.tag?(@bottle_tag, no_older_versions: true)
        end
```

Reply:
Needs to be negated, but otherwise should work.